client/src/pages/AdminLogin/index.jsx

```diff
@@ -1,7 +1,138 @@
-import React from "react";
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+
+import ADMIN_ACTIONS from '../../store/actions/adminLoginActions';
+import SNACKBAR_ACTIONS from '../../store/actions/snackbarActions';
+
+import useStyles from './styles';
+import Copyright from '../../components/Copyright';
+import loginAdmin from './apiCalls';
 
 const AdminLogin = () => {
-    return <></>;
+  const [helperText, setHelperText] = useState('');
```

Comment:
I think we should only be using redux, and no use states

Reply:
Fixed in `jwt-authentication-login-page` branch